src/Functions/map.cpp

```diff
@@ -26,6 +26,8 @@ namespace ErrorCodes
 {
     extern const int NUMBER_OF_ARGUMENTS_DOESNT_MATCH;
     extern const int ILLEGAL_TYPE_OF_ARGUMENT;
+    extern const int SIZES_OF_ARRAYS_DOESNT_MATCH;
```

Comment:
I fixed the typo in https://github.com/ClickHouse/ClickHouse/pull/46940 ... kindly rebase on master once the other PR is merged.